README.md

```diff
@@ -138,6 +137,49 @@ Below is a typical example of how this use case would take place:
 
 - Compute the new root for the vault’s rewards distributor, submit it, wait for the timelock (if any), accept the root, and let vault depositors claim their rewards according to the vault manager’s rewards re-distribution strategy.
 
+## Emergency
+
+### An enabled market is now considered unsafe
+
+If an enabled market is considered unsafe (e.g., risk too high), the curator/owner may want to disable this market in the following way:
+
+- Revoke the pending cap of the market with the `revokePendingCap` function (this can also be done by the guardian).
+- Set the cap of the market to 0 with the `submitCap` function.
+  To ensure that submit cap does not revert because of a pending cap, it is recommended to batch the two previous transactions, for example using the multicall function of MetaMorpho.
+- Withdraw all the supply of this market with the `reallocate` function.
+  If there is not enough liquidity on the market, remove the maximum available liquidity with the `reallocate` function, then put the market at the beginning of the withdraw queue (with the `updateWithdrawQueue` function).
+- Once all the supply has been removed from the market, the market can be removed from the withdraw queue with the `updateWithdrawQueue` function.
+
+### An enabled market reverts
+
+If an enabled market starts reverting, many of the vault functions would revert as well (because of the call to `totalAssets`). To turn the vault back to an operating state, the market must be forced removed by the owner/curator, who should follow these steps:
+
+- Revoke the pending cap of the market with the `revokePendingCap` function (this can also be done by the guardian).
+- Set the cap of the market to 0 with the `submitCap` function.
+  To ensure that submit cap does not revert because of a pending cap, it is recommended to batch the two previous transactions, for example using the multicall function of MetaMorpho.
+- Submit a removal of the market with the `submitMarketRemoval` function.
+- Wait for the timelock to elapse
+- Once the timelock has elapsed, the market can be removed from the withdraw queue with the `updateWithdrawQueue` function.
+
+Warning : Funds supplied in forced removed markets will be lost, this why only reverting market should be disabled this way (because funds supplied in such markets can be considered lost anyway).
```

Comment:
```suggestion
Warning : Funds supplied in forced removed markets will be lost, this is why only market expect to always revert should be disabled this way (because funds supplied in such markets can be considered lost anyway).
```